### ISSUE TITLE
@Optional Issue

### ISSUE THREAD

**Issue by GamerRealm** [vaperion/blade#12]
Hey so am trying to use @Optional from the example in the README.md and for some reason it doesn't work for me.
Am using the latest version. (1.2.10).

This is the code of the command:
```java
    @Command({"balance", "bal"})
    public static void balance(@Sender Player sender,
                               @Optional Player target) {
        if (target == null) {
            sender.sendMessage("Your balance is: $100");
        } else {
            sender.sendMessage(target.getName() + "'s balance is: $100");
        }
    }
```

This is in the Main class
```java
        Blade.of()
                .fallbackPrefix("fallbackPrefix")
                .containerCreator(BukkitCommandContainer.CREATOR)
                .binding(new BukkitBindings())
                .build()
                .register(BalanceCommand.class);
```

Issue:
![image](https://user-images.githubusercontent.com/42650369/140173126-9d9bd16c-0106-4b0f-b43b-c0693d151a3d.png)

**Comment by GitRowin** [comment]
I just ran into this issue as well, I'll look into it tomorrow.

**Comment by vaperion** [comment]
Fixed in 2.0.0.
